From ccb5bb112c01e26af8192ea7e1896bd51ca15ef2 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 31 May 2024 10:24:25 +0200
Subject: [PATCH 0887/1695] dts: st: add scmi-agent-id in scmi_regu node in
 stm32mp251.dtsi

After patch "core: scmi-server: regulator: use agent_id and channel_id
from parent", nodes with compatible "st,scmi-regulator-consumer" should at
least have a scmi-agent-id property and a scmi-channel-id.

Update stm32mp251.dtsi

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 53fe1a06107422a5be15fed386e2e289ca088830 dts: st: add scmi-agent-id in scmi_regu node in stm32mp251.dtsi]
Co-developed-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Signed-off-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I21df82d24903acd8e0c2e6603ed9b9585f532798
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/384364
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 22183563d..0389c1eb3 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -138,7 +138,7 @@
 		compatible = "st,scmi-regulator-consumer";
 		#address-cells = <1>;
 		#size-cells = <0>;
-
+		scmi-agent-id = <0>;
 		scmi-channel-id = <0>;
 
 		voltd-vddio1 {
-- 
2.47.2

